src/components/Carousel.js

```diff
@@ -115,6 +115,29 @@ class Carousel extends Component {
         this.destroyCarousel();
     }
 
+    setThumbsRef = e => {
+        this.thumbsRef = e;
+    }
+
+    setCarouselWrapperRef = e => {
+        this.carouselWrapperRef = e;
+    }
+
+    setListRef = e => {
+        this.list = e;
+    }
+
+    setItemsWrapperRef = e => {
+        this.itemsWrapperRef = e;
+    }
+
+    setItemsRef = (e, index) => {
+        if (!this.itemsRef) {
+            this.itemsRef = {};
```

Comment:
This was created as object but you are accessing as array in the next line. It's better to create as array `[]`...

Reply:
Ah yes, the variable behaves very much like an array. Will change this.

Reply:
It does because the key of the object matched. But it's wrong :)